submissions/ebojudahCherish/css/signUp.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```

Comment:
Kindly place this file inside `html-forms` folder. 
![image](https://user-images.githubusercontent.com/43061141/98438999-ba030f00-20ee-11eb-82b7-0de934857aae.png)